src/graphql/main/schema.graphql

```diff
@@ -929,7 +934,8 @@ input PriceInput {
 
 interface PriceInterface {
   base: SafeInt!
-  currencyUnit: String!
+  currencyUnit: CurrencyUnit!
```

Comment:
is this not used in prod already?

Reply:
I don't quite get why we want to return Major or Minor?  Why no already return Minor?

Reply:
this only a change for realtimePrice, I asked @UncleSamtoshi about this field and is not used also I am changing it because does not make sense to use `${currency}CENTS` for all currencies (specially for currencies that dont use CENTS as minor unit).

Reply:
I think the more important information here might be more which currency this is (`USD`/`EUR`/`CRC`) and less whether it's `MAJOR`/`MINOR`. I can't think of a case where we'd need `MAJOR` sometime soon... and if no-one else can either, then maybe we can deprecate this and re-introduce it when we need it again

Reply:
Add deprecation reason to currency unit https://github.com/GaloyMoney/galoy/pull/2414/commits/f48c847ae9cdff7666dd0d079aee0717b5b5633b